docs/root/configuration/listeners/listener_filters/original_dst_filter.rst

```diff
@@ -3,13 +3,42 @@
 Original Destination
 ====================
 
+Linux
+===============
+
 Original destination listener filter reads the SO_ORIGINAL_DST socket option set when a connection
 has been redirected by an iptables REDIRECT target, or by an iptables TPROXY target in combination
 with setting the listener's :ref:`transparent <envoy_v3_api_field_config.listener.v3.Listener.transparent>` option.
+
+Windows
+===============
+
+Original destination listener filter reads the SO_ORIGINAL_DST socket option set when a connection has been redirected by an
+`HNS <https://docs.microsoft.com/en-us/virtualization/windowscontainers/container-networking/architecture#container-network-management-with-host-network-service>`_
+policy applied to a container endpoint. For this filter to work the
+:ref:`traffic_direction <envoy_v3_api_field_config.listener.v3.Listener.traffic_direction>` must be set
+on the listener. This means that a separate listener is needed to handle inbound and outbound traffic.
+
+Redirection is not available for use with all types of network traffic. The types of packets that are supported for redirection are shown in the following list:
+
+* TCP
+* UDP
+* Raw UDPv4 without the header include option
+* Raw ICMP
+
+For more info see `Using Bind or Connect Redirection <https://docs.microsoft.com/en-us/windows-hardware/drivers/network/using-bind-or-connect-redirection>`_
+
+.. note::
+
+    At the time of writing (February 2021) the OS support for original destination is only available through the
+    `Windows insider program <https://insider.windows.com/en-us/for-developers>`_.
+    The feature will be fully supported in Windows Server 20H2
+    `Windows Server Release info <https://docs.microsoft.com/en-us/windows-server/get-started/windows-server-release-info>`_.
```

Comment:
Not in 20H2. It will be in the next Windows Server release. We don't have a public name for it yet.

Reply:
fixed in 2843f25025763c74a05673d5c618d9f83ce72e1c